server/middleware/one-time-code-auth/setUpOneTimeCode.ts

```diff
@@ -6,7 +6,7 @@ import RequestOneTimeCodeController from '../../routes/one-time-code-auth/Reques
 import setupOneTimeCodeEmailSent from './setupOneTimeCodeEmailSent'
 import OneTimeCodeEmailSentController from '../../routes/one-time-code-auth/OneTimeCodeEmailSentController'
 import setupVerifyOneTimeCode from './setupVerifyOneTimeCode'
-import setupStartAgain from './setupStartAgain'
```

Comment:
This function simply set the route for the start again page to render the view, with no complexities such as a controller etc. I was about to create a very similar function for setting up 'code no longer valid', but decided that as both were so simple (just setting up a route to a view) I'd do them both inline